test/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -133,21 +140,21 @@ public void createSynapseProjectTeam() throws SynapseException {
 
         // execute
         Study retStudy = studyService.createSynapseProjectTeam(TEST_USER_ID, study);
-
```

Comment:
Our unit tests should not have a dependency on Synapse. This is bad because (a) it makes our tests flaky (b) it means if Synapse is down for maintenance, this blocks our builds (c) it adds extra load on Synapse.

Please delete this test.